Open API File request for Translator Text API V3

kellyaltom opened the pull request:
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

kellyaltom committed:
Create readme.md

kellyaltom committed:
Delete readme.md

kellyaltom committed:
Create Readme.md

kellyaltom committed:
Create readme.md

kellyaltom committed:
Add files via upload

kellyaltom committed:
Update Readme.md

kellyaltom committed:
Update Readme.md

kellyaltom committed:
Update Readme.md

kellyaltom committed:
Update readme.md

kellyaltom committed:
Update readme.md

kellyaltom committed:
Update Readme.md

kellyaltom committed:
Create readme.md

kellyaltom committed:
Create Readme.md

kellyaltom committed:
Update Readme.md

kellyaltom committed:
Delete TextTranslationV3.json

kellyaltom committed:
Delete readme.md

kellyaltom committed:
Delete Readme.md

kellyaltom committed:
Add files via upload

kellyaltom committed:
Update readme.md

kellyaltom committed:
Update readme.md

kellyaltom committed:
Update readme.md

kellyaltom committed:
Rename Readme.md to Inof.md

kellyaltom committed:
Rename Inof.md to Info.md

kellyaltom committed:
Create TextAlignmentResults

kellyaltom committed:
Update TextAlignmentResults

kellyaltom committed:
Create TranslateOK

kellyaltom committed:
Rename TranslateOK to TranslateResult

kellyaltom committed:
Rename TranslateResult to TranslateResult.json

kellyaltom committed:
Add files via upload

AutorestCI commented:
# Automation for azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-go

AutorestCI commented:
# Automation for azure-sdk-for-ruby

azuresdkci commented:
Can one of the admins verify this patch?

AutorestCI commented:
# Automation for azure-sdk-for-node

AutorestCI commented:
# Automation for azure-sdk-for-java

kellyaltom commented:
I need to know if there is something wrong I  need to fix.

marstr reviewed (changes requested):
Our CI is failing because there is invalid JSON in this PR. Please fix that before we continue reviewing.

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/430223210#L570

kellyaltom committed:
Delete TextAlignmentResults

kellyaltom committed:
Delete TranslateResult.json

AutorestCI commented:
# Automation for azure-sdk-for-js

kellyaltom commented:
The error was in two files that I forgot to remove. It should work now.

kellyaltom commented:
The error was in a file that I forgot to remove. I deleted them.




________________________________
From: Martin Strobel <notifications@github.com>
Sent: Thursday, September 20, 2018 9:48 AM
To: Azure/azure-rest-api-specs
Cc: KellyAltom; Author
Subject: Re: [Azure/azure-rest-api-specs] Open API File request for Translator Text API V3 (#3943)


@marstr requested changes on this pull request.

Our CI is failing because there is invalid JSON in this PR. Please fix that before we continue reviewing.

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/430223210#L570<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Ftravis-ci.org%2FAzure%2Fazure-rest-api-specs%2Fjobs%2F430223210%23L570&data=02%7C01%7C%7Ca3db454c8b274e720c5b08d61f18f70d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636730589409486081&sdata=siEw4i5NPmbEz9LCUWAP%2BAOytguTCGskd%2FW9C6m6484%3D&reserved=0>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23pullrequestreview-157367679&data=02%7C01%7C%7Ca3db454c8b274e720c5b08d61f18f70d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636730589409486081&sdata=NpxEjdf5fsw9GIgxQCAEVeqp1Dw6VwE%2F%2BOc%2BCL0EUAE%3D&reserved=0>, or mute the thread<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOUo1H0W7t9TDdjcq_a0yeKi5vPaHC_wks5uc8b7gaJpZM4Wuotc&data=02%7C01%7C%7Ca3db454c8b274e720c5b08d61f18f70d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636730589409486081&sdata=fO1h2vge%2BpDugj04wZN2pIuhYyfOPcWNohjtHmxTE3k%3D&reserved=0>.

kellyaltom commented:
should the output file path be the same for all entries in the readme?


________________________________
From: Martin Strobel <notifications@github.com>
Sent: Thursday, September 20, 2018 9:48 AM
To: Azure/azure-rest-api-specs
Cc: KellyAltom; Author
Subject: Re: [Azure/azure-rest-api-specs] Open API File request for Translator Text API V3 (#3943)


@marstr requested changes on this pull request.

Our CI is failing because there is invalid JSON in this PR. Please fix that before we continue reviewing.

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/430223210#L570<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Ftravis-ci.org%2FAzure%2Fazure-rest-api-specs%2Fjobs%2F430223210%23L570&data=02%7C01%7C%7Ca3db454c8b274e720c5b08d61f18f70d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636730589409486081&sdata=siEw4i5NPmbEz9LCUWAP%2BAOytguTCGskd%2FW9C6m6484%3D&reserved=0>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23pullrequestreview-157367679&data=02%7C01%7C%7Ca3db454c8b274e720c5b08d61f18f70d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636730589409486081&sdata=NpxEjdf5fsw9GIgxQCAEVeqp1Dw6VwE%2F%2BOc%2BCL0EUAE%3D&reserved=0>, or mute the thread<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOUo1H0W7t9TDdjcq_a0yeKi5vPaHC_wks5uc8b7gaJpZM4Wuotc&data=02%7C01%7C%7Ca3db454c8b274e720c5b08d61f18f70d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636730589409486081&sdata=fO1h2vge%2BpDugj04wZN2pIuhYyfOPcWNohjtHmxTE3k%3D&reserved=0>.

kellyaltom committed:
Update readme.md

fixed characters before ## Java

kellyaltom commented:
fixed missing ``` before java section

marstr commented:
@AutorestCI rebuild go

marstr reviewed (changes requested):
Instead of adding a new README with AutoRest configuration in the v3.0 folder, you'll need to just modify the [README in the parent directory](https://github.com/Azure/azure-rest-api-specs/blob/acd9ddb21a2b94325d3f5ad823938b40469378a5/specification/cognitiveservices/data-plane/TextAnalytics/readme.md).

kellyaltom committed:
Create readme.md

kellyaltom commented:
Readme now in parent directory with autorest config.


________________________________
From: Martin Strobel <notifications@github.com>
Sent: Friday, September 21, 2018 1:37 PM
To: Azure/azure-rest-api-specs
Cc: KellyAltom; Author
Subject: Re: [Azure/azure-rest-api-specs] Open API File request for Translator Text API V3 (#3943)


@marstr requested changes on this pull request.

Instead of adding a new README with AutoRest configuration in the v3.0 folder, you'll need to just modify the README in the parent directory<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fblob%2Facd9ddb21a2b94325d3f5ad823938b40469378a5%2Fspecification%2Fcognitiveservices%2Fdata-plane%2FTextAnalytics%2Freadme.md&data=02%7C01%7C%7Cff9fbce8465f4b4a874408d62002163f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636731590658590194&sdata=dl3yDL9MJlOeJre%2Bd6LoosyNvEK10x2oWIraN04esSQ%3D&reserved=0>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23pullrequestreview-157850233&data=02%7C01%7C%7Cff9fbce8465f4b4a874408d62002163f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636731590658590194&sdata=Ct0zoG1UB3clWAxDGuz7ltr8%2FcaHLyNhhtJCIGbYePc%3D&reserved=0>, or mute the thread<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOUo1CfnvSlhkj_Q8uSoJGvYQOG-68-Iks5udU4YgaJpZM4Wuotc&data=02%7C01%7C%7Cff9fbce8465f4b4a874408d62002163f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636731590658590194&sdata=oKHJrlVEnH8MS2U%2B2GQTDMXUbEE1Pc%2FvCzR3LTbohOM%3D&reserved=0>.

kellyaltom commented:
I see the other errors about examples and am working on them.


________________________________
From: Martin Strobel <notifications@github.com>
Sent: Friday, September 21, 2018 1:37 PM
To: Azure/azure-rest-api-specs
Cc: KellyAltom; Author
Subject: Re: [Azure/azure-rest-api-specs] Open API File request for Translator Text API V3 (#3943)


@marstr requested changes on this pull request.

Instead of adding a new README with AutoRest configuration in the v3.0 folder, you'll need to just modify the README in the parent directory<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fblob%2Facd9ddb21a2b94325d3f5ad823938b40469378a5%2Fspecification%2Fcognitiveservices%2Fdata-plane%2FTextAnalytics%2Freadme.md&data=02%7C01%7C%7Cff9fbce8465f4b4a874408d62002163f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636731590658590194&sdata=dl3yDL9MJlOeJre%2Bd6LoosyNvEK10x2oWIraN04esSQ%3D&reserved=0>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23pullrequestreview-157850233&data=02%7C01%7C%7Cff9fbce8465f4b4a874408d62002163f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636731590658590194&sdata=Ct0zoG1UB3clWAxDGuz7ltr8%2FcaHLyNhhtJCIGbYePc%3D&reserved=0>, or mute the thread<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOUo1CfnvSlhkj_Q8uSoJGvYQOG-68-Iks5udU4YgaJpZM4Wuotc&data=02%7C01%7C%7Cff9fbce8465f4b4a874408d62002163f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636731590658590194&sdata=oKHJrlVEnH8MS2U%2B2GQTDMXUbEE1Pc%2FvCzR3LTbohOM%3D&reserved=0>.

marstr commented on specification/cognitiveservices/data-plane/TranslatorText/v3.0/readme.md:
````diff
@@ -0,0 +1,120 @@
+
+``` yaml
+
+tag: release_3_0
````
It's okay to have a README in this directory, but it should contain no AutoRest configuration. Please delete this file, or completely replace its contents.

marstr commented on specification/cognitiveservices/data-plane/TranslatorText/readme.md:
````diff
@@ -0,0 +1,120 @@
+These settings apply only when `--tag=release_1_0 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'release_1_0' && $(go)
````
Copy/paste error? Please update all `release_1_0` to `release_3_0`

marstr commented on specification/cognitiveservices/data-plane/TranslatorText/Info.md:
```diff
@@ -0,0 +1,4 @@
+# Microsoft Translator API V3
+API version 3 only. Translator V2 is depracated.
```
[nit] "depracated" should be "deprecated" here and in the line below.

kellyaltom committed:
Add files via upload

kellyaltom committed:
Update readme.md

kellyaltom committed:
Update readme.md

kellyaltom committed:
Update Info.md

kellyaltom commented:
added examples in request, added maximum to all array element lengths.

marstr reviewed (dismissed):
Looks like we still have some validation errors:
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/432730131#L605

kellyaltom commented:
Do I need to use the tag 'x-ms-examples' or can I just use 'example' to show examples in the 200 success case.




________________________________
From: Martin Strobel <notifications@github.com>
Sent: Tuesday, September 25, 2018 10:48 AM
To: Azure/azure-rest-api-specs
Cc: KellyAltom; Author
Subject: Re: [Azure/azure-rest-api-specs] Open API File request for Translator Text API V3 (#3943)


@marstr requested changes on this pull request.

Looks like we still have some validation errors:
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/432730131#L605<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Ftravis-ci.org%2FAzure%2Fazure-rest-api-specs%2Fjobs%2F432730131%23L605&data=02%7C01%7C%7C0a4d5fe7ba27484a7a0d08d6230f0c25%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636734944859574521&sdata=2Cmny0Ein2C7JNheNmnawG0eK7k31Ig8SsdwOdIkQl8%3D&reserved=0>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23pullrequestreview-158665523&data=02%7C01%7C%7C0a4d5fe7ba27484a7a0d08d6230f0c25%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636734944859574521&sdata=eRPoCAf7AhYyDLJg65NottfKdMfDnImWES%2Bwr3eQs6o%3D&reserved=0>, or mute the thread<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOUo1LHElAf4-PyhrV3i9yqab8vxvhhdks5uemxTgaJpZM4Wuotc&data=02%7C01%7C%7C0a4d5fe7ba27484a7a0d08d6230f0c25%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636734944859574521&sdata=DI5eQ2NAWbWKa8FnMRp3b4qHZ9j%2FubFAX2IxiMgenho%3D&reserved=0>.

kellyaltom committed:
Create TranslateSuccess.json

kellyaltom committed:
Update TranslateSuccess.json

kellyaltom committed:
Update TranslateSuccess.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom commented:
Hi Martin, can you help me understand what the errors are? I read the log files but can't make sense of them.

kellyaltom committed:
Rename readme.md to Info.md

kellyaltom reviewed (commented):
I have made the fixes

yangyuan commented on specification/cognitiveservices/data-plane/TranslatorText/Info.md:
```diff
@@ -0,0 +1,4 @@
+# Microsoft Translator API V3
```
Any specific reason we need this file?
This is specifications repo, this kind of announcement should be in documents (or swaggers "info"."description" field).

yangyuan commented on specification/cognitiveservices/data-plane/TranslatorText/readme.md:
````diff
@@ -0,0 +1,120 @@
+These settings apply only when `--tag=release_3_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_3_0'
+input-file: v3.0/TranslatorText.json
````
According to the short document in root page: https://github.com/Azure/azure-rest-api-specs
There should be either a `stable` folder or `preview` folder.
`stable/v3.0/TranslatorText.json`
`preview/v3.0/TranslatorText.json`

yangyuan commented on specification/cognitiveservices/data-plane/TranslatorText/readme.md:
```diff
@@ -0,0 +1,120 @@
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-node
```
I didn't see `azure-sdk-for-node` and `azure-sdk-for-ruby` configs...
I think you need to add them or remove these two lines

yangyuan commented on specification/cognitiveservices/data-plane/TranslatorText/readme.md:
````diff
@@ -0,0 +1,120 @@
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-cognitiveservices-vision-face/azure/cognitiveservices/translatortext
````
azure-cognitiveservices-**vision-face** ?

yangyuan commented on specification/cognitiveservices/data-plane/TranslatorText/readme.md:
````diff
@@ -0,0 +1,120 @@
+  - tag: release_3_0
+```
+
+### Tag: release_1_0 and go
````
Should be `release_3_0 and go`?

yangyuan commented on specification/cognitiveservices/data-plane/TranslatorText/v3.0/Info.md:
```diff
@@ -0,0 +1,2 @@
+# Cognitive Services Translator Text SDK
```
Please remove this file.

yangyuan commented on specification/cognitiveservices/data-plane/TranslatorText/v3.0/Examples/TranslateSuccess.json:
```diff
@@ -0,0 +1,10 @@
+[
```
The folder name should be in lower case `examples`.

yangyuan commented:
Hi @kellyaltom,

I saw a few small mistakes. I pointed them out in the review, hope they will be helpful for you to pass the build.

kellyaltom committed:
Delete Info.md

kellyaltom committed:
Delete Info.md

kellyaltom committed:
Create TranslatorText.json

kellyaltom committed:
Create translateSuccess.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Delete TranslateSuccess.json

kellyaltom committed:
Delete TranslatorText.json

kellyaltom committed:
Update readme.md

kellyaltom committed:
Update readme.md

kellyaltom committed:
Update TranslatorText.json

kellyaltom commented:
Added fixes

kellyaltom committed:
Create transliterateSuccess

kellyaltom committed:
Create dictionaryLookupSuccess

kellyaltom committed:
Create dictionaryExampleSuccess

kellyaltom committed:
Create detectSuccess

kellyaltom committed:
Rename detectSuccess to detectSuccess.json

kellyaltom committed:
Rename dictionaryExampleSuccess to dictionaryExampleSuccess.json

kellyaltom committed:
Rename dictionaryLookupSuccess to dictionaryLookupSuccess.json

kellyaltom committed:
Rename transliterateSuccess to transliterateSuccess.json

kellyaltom committed:
Create breakSentenceSuccess.json

kellyaltom committed:
Create languagesSuccess

kellyaltom committed:
Update TranslatorText.json

kellyaltom commented:
Fixed examples for 200 success

kellyaltom committed:
Update TranslatorText.json

changed Desc. string.

kellyaltom committed:
Delete TranslatorText.json

kellyaltom committed:
Create TranslatorText.json

kellyaltom committed:
Update readme.md

kellyaltom commented:
Added Stable directory to the path

kellyaltom committed:
Rename languagesSuccess to languagesSuccess.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update translateSuccess.json

kellyaltom committed:
Update translateSuccess.json

kellyaltom committed:
Update translateSuccess.json

kellyaltom committed:
Update translateSuccess.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update TranslatorText.json

marstr commented:
Howdy @kellyaltom, what's the current status of this PR?

kellyaltom committed:
Update TranslatorText.json

Fixed some errors

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update TranslatorText.json

Fixed errors in text input examples

kellyaltom committed:
Update translateSuccess.json

fixed example

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update translateSuccess.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update translateSuccess.json

fixed an error

nschonni commented on specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json:
```diff
@@ -0,0 +1,1105 @@
+      "email": "translator@microsoft.com",
+      "url": "https://www.microsoft.com/translator"
+    },
+    "description": "# Introduction\n\nThe Microsoft Translator Text API provides a JSON-based Web API. It provides:\n\n  * Translation between any supported languages to any other supported language.\n  * Translation to multiple languages in one request.\n  * Transliteration to convert text from one script to another script of the same language.\n  * Language detection, translation, and translite ...
```
beforethe -> before the

nschonni commented on specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json:
```diff
@@ -0,0 +1,1105 @@
+          },
+          {
+            "name": "text",
+            "description": "# Request Body\nThe body of the request is a JSON array. Each array element is a JSON object with a string property named `Text`, which represents the term to lookup.\nThe following limitations apply:\n* The array can have at most 10 elements.\n* The text value of an array element cannot exceed 100 characters including spaces.\n\n# Response Body\nA successful response  ...
```
Conjuctions -> Conjunctions

nschonni commented on specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json:
```diff
@@ -0,0 +1,1105 @@
+    },
+    "DictionaryExampleTextInput": {
+      "type": "object",
+      "description": "Text needed for a dictionary example rquest ",
```
rquest -> request

nschonni commented on specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json:
```diff
@@ -0,0 +1,1105 @@
+    "application/json"
+  ],
+  "paths": {
+    "/Breaksentence": {
```
Should this be lowercase `breaksentence` or camel case `BreakSentence` instead?

kellyaltom committed:
Update transliterateSuccess.json

kellyaltom committed:
Update dictionaryLookupSuccess.json

kellyaltom committed:
Update dictionaryExampleSuccess.json

kellyaltom committed:
Update dictionaryExampleSuccess.json

kellyaltom committed:
Update breakSentenceSuccess.json

kellyaltom committed:
Update detectSuccess.json

kellyaltom committed:
Update transliterateSuccess.json

kellyaltom committed:
Update languagesSuccess.json

kellyaltom commented:
Added all example json files

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Squashed commits to single commit for several changes
Update transliterateSuccess.json

Update languagesSuccess.json

Update languagesSuccess.json

Update TranslatorText.json

Update TranslatorText.json

Update TranslatorText.json

Added securityDefinitions

Removed Auth parameters from breakSentence Method

Removed Auth from Method

Removed auth from method

Update TranslatorText.json

Removed Ocp-Apim-Subscription-key and Authorization from each method because it is in the header of the file.

kellyaltom committed:
Update transliterateSuccess.json

kellyaltom committed:
Update languagesSuccess.json

kellyaltom committed:
Update languagesSuccess.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom commented:
This pull request is passing all tests but is blocked from merging. I have resolved all reviews. Can someone tell me what I need to do to finish this request?

Thanks,
Kelly
________________________________
From: Nick Schonning <notifications@github.com>
Sent: Wednesday, December 5, 2018 6:58 PM
To: Azure/azure-rest-api-specs
Cc: KellyAltom; State change
Subject: Re: [Azure/azure-rest-api-specs] Open API File request for Translator Text API V3 (#3943)


@nschonni commented on this pull request.

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r239313828&data=02%7C01%7C%7Cdb9e0793f843423a4f6a08d65b26b212%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636796619079544214&sdata=zxFOthwab774lLg4cKfNjMWS%2BsovKIwk23wkX%2F3x31g%3D&reserved=0>:

> +            "name": "Ocp-Apim-Subscription-Key",
+            "description": "This is used to pass a key for auth. If you are passing a token for auth then use the previous header auth option. **ONE OF THESE METHODS MUST BE USED.**",
+            "in": "header",
+            "type": "string",
+            "required": false
+          },
+          {
+            "name": "X-ClientTraceId",
+            "description": "A client-generated GUID to uniquely identify the request. Note that you can omit this header if you include the trace ID in the query string using a query parameter named ClientTraceId.",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "text",
+            "description": "# Request Body\nThe body of the request is a JSON array. Each array element is a JSON object with a string property named `Text`, which represents the term to lookup.\nThe following limitations apply:\n* The array can have at most 10 elements.\n* The text value of an array element cannot exceed 100 characters including spaces.\n\n# Response Body\nA successful response is a JSON array with one result for each string in the input array. A result object includes the following properties:\n* `normalizedSource`- A string giving the normalized form of the source term. For example, if the request is \"JOHN\", the normalized form will be \"john\". The content of this field becomes the input to lookup examples.\n* `displaySource`- A string giving the source term in a form best suited for end-user display. For example, if the input is \"JOHN\", the display form will reflect the usual spelling of the name- \"John\".\n* `translations`- A list of translations for the source term. Each element of the list is an object with the following properties:\n  * `normalizedTarget`- A string giving the normalized form of this term in the target language. This value should be used as input to lookup examples.\n  * `displayTarget`- A string giving the term in the target language and in a form best suited for end-user display. Generally, this will only differ from the `normalizedTarget` in terms of capitalization. For example, a proper noun like \"Juan\" will have `normalizedTarget = \"juan\"` and `displayTarget = \"Juan\"`.\n  * `posTag`- A string associating this term with a part-of-speech tag.\n  \n  | Tag name  | Description       |\n  | --------- | ----------------  |\n  | ADJ       | Adjectives        |\n  | ADV       | Adverbs           |\n  | CONJ      | Conjuctions       |\n  | DET       | Determiners       |\n  | MODAL     | Verbs             |\n  | NOUN      | Nouns             |\n  | PREP      | Prepositions      |\n  | PRON      | Pronouns          |\n  | VERB      | Verbs             |\n  | OTHER     | Other             |\n  \n  As an implementation note, these tags were determined by part-of-speech tagging the English side, and then taking the most frequent tag for each source/target pair. So if people frequently translate a Spanish word to a different part-of-speech tag in English, tags may end up being wrong (with respect to the Spanish word).\n    * `confidence`- A value between 0.0 and 1.0 which represents the \"confidence\" (or perhaps more accurately, \"probability in the training data\") of that translation pair. The sum of confidence scores for one source word may or may not sum to 1.0.\n    * `prefixWord-` A string giving the the word to display as a prefix of the translation. Currently, this is the gendered determiner of nouns, in languages that have gendered determiners. For example, the prefix of the Spanish word \"mosca\" is \"la\", since \"mosca\" is a feminine noun in Spanish. This is only dependent on the translation, and not on the source. If there is no prefix, it will be the empty string.\n    * `backTranslations-` A list of \"back translations\" of the target. For example, source words that the target can translate to. The list is guaranteed to contain the source word that was requested (e.g., if the source word being looked up is \"fly\", then it is guaranteed that \"fly\" will be in the `backTranslations` list). However, it is not guaranteed to be in the first position, and often will not be. Each element of the `backTranslations` list is an object described by the following properties-\n      * `normalizedText-` A string giving the normalized form of the source term that is a back-translation of the target. This value should be used as input to lookup examples.\n      * `displayText-` A string giving the source term that is a back-translation of the target in a form best suited for end-user display.\n      * `numExamples-` An integer representing the number of examples that are available for this translation pair. Actual examples must be retrieved with a separate call to lookup examples. The number is mostly intended to facilitate display in a UX. For example, a user interface may add a hyperlink to the back-translation if the number of examples is greater than zero and show the back-translation as plain text if there are no examples. Note that the actual number of examples returned by a call to lookup examples may be less than `numExamples`, because additional filtering may be applied on the fly to remove \"bad\" examples.\n      * `frequencyCount-` An integer representing the frequency of this translation pair in the data. The main purpose of this field is to provide a user interface with a means to sort back-translations so the most frequent terms are first.\n      \nNote - If the term being looked up does not exist in the dictionary, the response is 200 (OK) but the `translations` list is an empty list.\n  \n# Response Header\nX-RequestId - Value generated by the service to identify the request. It is used for troubleshooting purposes.\n",


Conjuctions -> Conjunctions

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r239313909&data=02%7C01%7C%7Cdb9e0793f843423a4f6a08d65b26b212%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636796619079544214&sdata=xDvIqQ4JuHKd929XNpWhIfgOnR8ZBQIRsRZ0lPIGsak%3D&reserved=0>:

> +    "DictionaryLookupTextInput": {
+      "type": "object",
+      "description": "Text needed for a dictionary lookup request ",
+      "properties": {
+        "text": {
+          "type": "string",
+          "maximum": 100
+        }
+      },
+      "example": {
+        "text": "Bug"
+      }
+    },
+    "DictionaryExampleTextInput": {
+      "type": "object",
+      "description": "Text needed for a dictionary example rquest ",


rquest -> request

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r239314158&data=02%7C01%7C%7Cdb9e0793f843423a4f6a08d65b26b212%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636796619079544214&sdata=0SFCLLpAvSiPYF%2FBOs5tzML1nRDP%2FJPTu%2FfuqCOPMOM%3D&reserved=0>:

> +        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "host": "api.cognitive.microsofttranslator.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Breaksentence": {


Should this be lowercase breaksentence or camel case BreakSentence instead?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23pullrequestreview-182066415&data=02%7C01%7C%7Cdb9e0793f843423a4f6a08d65b26b212%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636796619079544214&sdata=bLC0tsFvNhNCVfMQmy51DgZxVkRAAj79dkNpyCNy7U0%3D&reserved=0>, or mute the thread<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOUo1JCxNEJYgXKO9XYrsOQ_J9L20o3aks5u2IfSgaJpZM4Wuotc&data=02%7C01%7C%7Cdb9e0793f843423a4f6a08d65b26b212%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636796619079544214&sdata=vxoe4uFlhYuy1spMX1P2KG%2F7YK7dXKwkpCFpi%2FXXBMw%3D&reserved=0>.

kellyaltom commented:
Hi Yang,

I fixed all of the items in your review. Can you review them so the PR can merge.

Thanks,
Kelly
________________________________
From: Yang Yuan <notifications@github.com>
Sent: Thursday, September 27, 2018 11:38 PM
To: Azure/azure-rest-api-specs
Cc: KellyAltom; State change
Subject: Re: [Azure/azure-rest-api-specs] Open API File request for Translator Text API V3 (#3943)


@yangyuan requested changes on this pull request.

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/Info.md<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r221152789&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246274105&sdata=J4r4IKiwFPgBsjg1eJzaxmynSIccXToOx0kdzUCYqe0%3D&reserved=0>:

> @@ -0,0 +1,4 @@
+# Microsoft Translator API V3


Any specific reason we need this file?
This is specifications repo, this kind of announcement should be in documents (or swaggers "info"."description" field).

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/readme.md<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r221153166&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246274105&sdata=5OFSUa1QcOOvGzFaPnDlaGUEAjHwzxUlnY4vVj1pLNQ%3D&reserved=0>:

> +The current release is `release_3_0`.
+
+``` yaml
+
+tag: release_3_0
+add-credentials: true
+openapi-type: data-plane
+```
+
+# Releases
+
+### Release 3.0
+These settings apply only when `--tag=release_3_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_3_0'
+input-file: v3.0/TranslatorText.json


According to the short document in root page: https://github.com/Azure/azure-rest-api-specs<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246274105&sdata=lDlI%2FzRllqegX4kB3ZzUYmKgyprNKK2vmFDEvXYyfaM%3D&reserved=0>
There should be either a stable folder or preview folder.
stable/v3.0/TranslatorText.json
preview/v3.0/TranslatorText.json

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/readme.md<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r221153307&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246274105&sdata=BFFG7JuZ4Tw4si0ikRGJs1FtiMqQuHr8hL%2F3oYc0M0M%3D&reserved=0>:

> +
+``` yaml $(tag) == 'release_3_0'
+input-file: v3.0/TranslatorText.json
+```
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-node


I didn't see azure-sdk-for-node and azure-sdk-for-ruby configs...
I think you need to add them or remove these two lines

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/readme.md<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r221153415&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246274105&sdata=okKPGbKevGnu4ypnEyxMyO9QDN8Qxdw49RJ%2B4WCqD6c%3D&reserved=0>:

> +Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  license-header: MICROSOFT_MIT_NO_VERSION
+  add-credentials: true
+  payload-flattening-threshold: 2
+  namespace: azure.cognitiveservices.translatortext
+  package-name: azure-cognitiveservices-translatortext
+  clear-output-folder: true
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-cognitiveservices-vision-face/azure/cognitiveservices/translatortext


azure-cognitiveservices-vision-face ?

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/readme.md<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r221153462&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246430354&sdata=2KlUYQ%2F2TvrGN7cE3bifzgLR5KF0yE60kfTYEOV6nfc%3D&reserved=0>:

> +
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: translatortext
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: release_3_0
+```
+
+### Tag: release_1_0 and go


Should be release_3_0 and go?

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/v3.0/Info.md<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r221153491&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246430354&sdata=gpdg6wmg5B%2Fj6d7t4O%2BsBq33LB6gcYcwarMSYkGQEiw%3D&reserved=0>:

> @@ -0,0 +1,2 @@
+# Cognitive Services Translator Text SDK


Please remove this file.

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/v3.0/Examples/TranslateSuccess.json<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r221153563&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246430354&sdata=3f5WKH9gFzbOws4XyCQZ6r9SQ%2BD4O42GyMQyVUYP2Rg%3D&reserved=0>:

> @@ -0,0 +1,10 @@
+[


The folder name should be in lower case examples.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23pullrequestreview-159733068&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246430354&sdata=NOH0sODAKLLE9Bd781l3%2BVp4TJqTc%2FGG%2BBLTkZDI3v4%3D&reserved=0>, or mute the thread<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOUo1PfRdu4Sjy-RxC8ZqxhlgUChJK1dks5ufcPzgaJpZM4Wuotc&data=02%7C01%7C%7Ce1cebe57c7bb42876fea08d6250d0969%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636737135246430354&sdata=3lCOCXsXY4k4QCnkIH8k2B9T0bYYQkW%2BN3GdiCrQpF0%3D&reserved=0>.

yangyuan reviewed (changes requested):
Looks good except authentication headers.
Please send the swagger to "Azure API Review Board Core <azureapirbcore@microsoft.com>" for design review and cc me (alias yuanyang)
Please do not expose internal email threads.

FYI, only ADX team can merge, and they will do a full review to make sure this fits azure standards.

kellyaltom committed:
Update TranslatorText.json

Added securityDefinitions

kellyaltom commented on specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json:
```diff
@@ -0,0 +1,1105 @@
+            "required": false
+          },
+          {
+            "name": "Ocp-Apim-Subscription-Key",
```
Added security definitions

yangyuan replied:
You also need to remove these headers from operations.
There will effect the behavior of generated SDK.

Have you tested the generated SDK?

kellyaltom commented:
Ok I will remove, I was not sure if they should stay or not. One method does not require security, Languages.

I did not know the SDKs were available for testing before the pull request was completed. Where are they at? I will test them.

Thanks for your help.

Kelly
________________________________
From: Yang Yuan <notifications@github.com>
Sent: Friday, December 7, 2018 3:44 PM
To: Azure/azure-rest-api-specs
Cc: KellyAltom; State change
Subject: Re: [Azure/azure-rest-api-specs] Open API File request for Translator Text API V3 (#3943)


@yangyuan commented on this pull request.

________________________________

In specification/cognitiveservices/data-plane/TranslatorText/stable/v3.0/TranslatorText.json<https://nam05.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r239977264&data=02%7C01%7C%7C53e5cad55a6f416f0f6608d65c9df742%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636798230851405938&sdata=bz1COWLBOdJnUmUjCF307ImaQPEh4mycjg5C7y3QlJ0%3D&reserved=0>:

> +          {
+            "name": "Script",
+            "description": "Script identifier of the script used by the input text. If a script is not specified, the default script of the language will be assumed.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "Authorization",
+            "description": "This is used to pass an authentication token. If you are passing a key and not a token for authentication, then use the other method in the header. **ONE OF THESE METHODS MUST BE USED.**",
+            "in": "header",
+            "type": "string",
+            "required": false
+          },
+          {
+            "name": "Ocp-Apim-Subscription-Key",


You also need to remove these headers from operations.
There will effect the behavior of generated SDK.

Have you tested the generated SDK?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://nam05.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F3943%23discussion_r239977264&data=02%7C01%7C%7C53e5cad55a6f416f0f6608d65c9df742%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636798230851405938&sdata=bz1COWLBOdJnUmUjCF307ImaQPEh4mycjg5C7y3QlJ0%3D&reserved=0>, or mute the thread<https://nam05.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOUo1P-I4GWsik8UcqrRXMEHRUrohcUyks5u2v1sgaJpZM4Wuotc&data=02%7C01%7C%7C53e5cad55a6f416f0f6608d65c9df742%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636798230851405938&sdata=8sgOPlBehxxaKF6R35v%2BNeLl51uRG2ZRGY6jX7n8daU%3D&reserved=0>.

Commit:
Removed Auth parameters from breakSentence Method

Commit:
Removed Auth from Method

Commit:
Removed auth from method

kellyaltom committed:
Update TranslatorText.json

Removed Ocp-Apim-Subscription-key and Authorization from each method because it is in the header of the file.

Commit:
Merge branch 'master' of https://github.com/kellyaltom/azure-rest-api-specs

Commit:
Merge branch 'master' of https://github.com/kellyaltom/azure-rest-api-specs

kellyaltom committed:
Made changes after Swagger review

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Swagger editor allows enum without double quotes.

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
final commit

kellyaltom committed:
Update TranslatorText.json

kellyaltom committed:
Merge branch 'master' of https://github.com/kellyaltom/azure-rest-api-specs

kellyaltom committed:
Update languagesSuccess.json

kellyaltom committed:
This is the 1st commit message:

conflict resolution

Update TranslatorText.json

conflict
